test_project/test_app/tests/test_user.py

```diff
@@ -63,7 +63,7 @@ def test_get_logged_in_user(self):
 
     def test_user_can_sign_up(self):
         url = reverse("sign_up")
-        password = base64.encodestring("testtest")
+        password = base64.encodebytes(b"testtest").decode()
```

Comment:
it's not much code, but maybe we want to create a short helper method for this since we do it a lot?